scripts/launch-xrd

```diff
@@ -250,18 +243,15 @@ while [[ $# -gt 0 ]]; do
            shift
```

Comment:
No need to handle name/privileged explicitly anymore?

Reply:
Not handling name anymore